calculator/src/components/display.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import './css/button.css'
+
+const Display = ({state, handleNumberClick}) => {
+    
+        return(
+         <div className="display">
+         <h2>{state.displayValue}</h2>
+         <div className="AllNumbers">
+            <button className="0" value="0" onClick={(e) =>handleNumberClick(e)}>0</button>
```

Comment:
You shouldn't give a different `className` to each button individually. If you are doing this for CSS purposes, you can give your number buttons a `className` (let's say `numbers`), and your operator buttons a `className` of `operators`, and maybe your zero would have an id of `zero` since it's the only button that spans 2 columns so it deserves individual attention, yet it's still number as well, so it should have a className `numbers` in addition to the id of `zero`.